app/javascript/packs/pages/edit_provider/EditProviderForm.jsx

```diff
@@ -25,6 +25,10 @@ const useStyles = makeStyles((theme) => ({
     fontSize: "14px",
     marginBottom: "50px",
   },
+  formLabel: {
```

Comment:
Do you think this change applies globally or is it specific to this page? If it's global, we can drop it in `StyledFormLabel`

Reply:
I think it's just specific to the provider form pages! I took a look at the other instances when StyledFormLabel is used and they look fine to me. Here are screenshots of the other locations where it's used. Let me know what you think :) 

![image](https://user-images.githubusercontent.com/32404681/78414243-1ab5f080-75d0-11ea-90e8-d01584fcd53d.png)
![image](https://user-images.githubusercontent.com/32404681/78414247-1f7aa480-75d0-11ea-9376-c0fb91124ba0.png)